common/src/main/java/com/graphicsfuzz/common/util/PipelineInfo.java

```diff
@@ -251,6 +251,35 @@ public void removeUniformBinding(String uniformName) {
     }
   }
 
+  public int appendValueToUniform(String uniformName, Number value) {
```

Comment:
Can you add a JavaDoc comment for this method, in particular explaining what the returned value means?

Reply:
Also: please add tests to PipelineInfoTest for each of the public methods that you have added or changed.

Reply:
Added the JavaDoc comment and the tests.